exercises/saddle-points/saddle_points_test.py

```diff
@@ -20,6 +20,10 @@ def test_identify_single_saddle_point(self):
     def test_empty_matrix_has_no_saddle_points(self):
         self.assertEqual(saddle_points([]), set())
 
+    def test_matrix_with_one_elem_has_single_saddle_point(self):
+        matrix = [[1]]
+        self.assertEqual(saddle_points(matrix), set([(0, 0)]))
+
```

Comment:
This test is not canonical, but I think it's a good one. Might not be a bad idea to create another PR in problem-specifications to add it there too, but I don't think this PR needs to be held up waiting on that.